cmd/cloud/dashboard.go

```diff
@@ -0,0 +1,152 @@
+				PerPage:        model.AllPerPage,
+				IncludeDeleted: false,
+			})
+			if err != nil {
+				return errors.Wrap(err, "failed to query clusters")
+			}
+
+			clusterCount := len(clusters)
+			var clusterStableCount int
+			for _, cluster := range clusters {
+				if cluster.State == model.ClusterStateStable {
+					clusterStableCount++
+				} else {
+					unstableList = append(unstableList, fmt.Sprintf("Cluster: %s (%s)", cluster.ID, cluster.State))
+				}
+			}
+
+			table.Append([]string{
+				"Cluster",
+				toStr(clusterCount),
+				toStr(clusterStableCount),
+				toStr(clusterCount - clusterStableCount),
+			})
+
+			// Installations
+			installations, err := client.GetInstallations(&model.GetInstallationsRequest{
+				Page:           0,
+				PerPage:        model.AllPerPage,
+				IncludeDeleted: false,
+			})
+			if err != nil {
+				return errors.Wrap(err, "failed to query installations")
+			}
+
+			installationCount := len(installations)
+			var installationStableCount int
+			for _, installation := range installations {
+				if installation.State == model.InstallationStateStable {
+					installationStableCount++
+				} else {
+					unstableList = append(unstableList, fmt.Sprintf("Installation: %s (%s)", installation.ID, installation.State))
+				}
+			}
+
+			table.Append([]string{
+				"Installation",
+				toStr(installationCount),
+				toStr(installationStableCount),
+				toStr(installationCount - installationStableCount),
+			})
+
+			clusterInstallations, err := client.GetClusterInstallations(&model.GetClusterInstallationsRequest{
+				Page:           0,
+				PerPage:        model.AllPerPage,
+				IncludeDeleted: false,
+			})
+			if err != nil {
+				return errors.Wrap(err, "failed to query clusters")
+			}
+
+			// Cluster Installations
+			clusterInstallationCount := len(clusterInstallations)
+			var clusterInstallationStableCount int
+			for _, clusterInstallation := range clusterInstallations {
+				if clusterInstallation.State == model.ClusterInstallationStateStable {
+					clusterInstallationStableCount++
+				} else {
+					unstableList = append(unstableList, fmt.Sprintf("Cluster Installation: %s (%s)", clusterInstallation.ID, clusterInstallation.State))
+				}
+			}
+
+			table.Append([]string{
+				"Cluster Installation",
+				toStr(clusterInstallationCount),
+				toStr(clusterInstallationStableCount),
+				toStr(clusterInstallationCount - clusterInstallationStableCount),
+			})
+
+			table.Render()
+			renderedDashboard := fmt.Sprintf("\n### CLOUD DASHBOARD\n\n")
+			renderedDashboard += tableString.String()
+			for _, entry := range unstableList {
+				renderedDashboard += fmt.Sprintf("%s\n", entry)
+			}
+			if len(unstableList) != 0 {
+				renderedDashboard += "\n"
+			}
+
+			for i := refreshSeconds; i > 0; i-- {
+				fmt.Fprintf(writer, "%s\nUpdating in %d seconds...\n", renderedDashboard, i)
+				time.Sleep(time.Second)
+			}
+		}
+	},
+}
+
+func toStr(i int) string {
+	return fmt.Sprintf("%d", i)
```

Comment:
I believe that a more canonical way of doing it is using `Itoa`
```suggestion
	return strconv.Itoa(i)
```